source/Cosmos.HAL2/Cosmos.HAL2.csproj

```diff
@@ -5,10 +5,11 @@
         <GenerateDocumentationFile>true</GenerateDocumentationFile>
         <RootNamespace>Cosmos.HAL</RootNamespace>
         <Configurations>Debug;Release;TEST</Configurations>
+        <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
```

Comment:
Can you please clean up this csproj file? Unsafe code should be uniformly enabled.